docs/specs/config.yml

```diff
@@ -241,4 +241,15 @@ inputs:
       docs/toc.md: /absolute/path
 outputs:
   build.log: |
-    ["error","invalid-redirection","The docs/toc.md shouldn't belong to redirections since it's a TableOfContents"]
+    ["error","invalid-redirection","The 'docs/toc.md' shouldn't belong to redirections since it's a TableOfContents"]
+---
+# Conflicted redirection file
+inputs:
+  docfx.yml: |
+    redirections:
+      docs/a.md: /absolute/path1
+    redirectionsWithoutDocumentId:
```

Comment:
Maybe we could **prefix** the URL with `!` to mean that it should also redirect document id?

Reply:
!/absolute/path1? not really easy to understand

---
In reply to: [196650230](https://github.com/dotnet/docfx/pull/2904#discussion_r196650230) [](ancestors = 196650230)